app/src/ui/diff/diff-line-gutter.tsx

```diff
@@ -149,8 +168,17 @@ export class DiffLineGutter extends React.Component<IDiffGutterProps, {}> {
   private getLineClass(): string {
     const lineClass = this.getLineClassName()
     const selectedClass = this.isIncluded() ? selectedLineClass : null
-
-    return classNames('diff-line-gutter', lineClass, selectedClass)
+    const hoverClass = this.state.hover ? hoverCssClass : null
+
+    return classNames(
+      'diff-line-gutter',
+      lineClass,
+      selectedClass,
+      hoverClass,
```

Comment:
You can remove the `hoverClass` variable and merge this into the object on the next line:

```js
{
  'read-only': this.props.readOnly,
  [hoverCssClass]: this.state.hover,
}
```

Reply:
That's a neat trick! I'm doing some follow up stuff for this and I'll try to get into the habit of doing this